.github/workflows/ci.yml

```
@@ -0,0 +1,57 @@
+            name: WordPress Plugin Check
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 0 but found 12

(indentation)